samples/kvsWebRTCClientMaster.c

```diff
@@ -343,16 +331,16 @@ PVOID sendAudioPackets(PVOID args)
     return (PVOID)(ULONG_PTR) retStatus;
 }
 
-PVOID sampleReceiveAudioFrame(PVOID args)
+PVOID sampleReceiveVideoFrame(PVOID args)
```

Comment:
This looks to be generic enough for any type. Should be make it agnostic of type?

Reply:
the routine we assign it to is pSampleConfiguration->receiveAudioVideoSource. User can assign any routine to receiveAudioVideoSource to setup which transceivers to receive.

Reply:
OK